### PR TITLE
reset trigger message when everything is ok

### PR DESCRIPTION
**Pull request by h00dy**

### CODE REVIEW

**Comment by pnuckowski** [comment]
👍

File: alamo_alerter/alerter.py [hunk 1]
```diff
@@ -153,6 +153,10 @@ def execute(self, check):
             # to that value
             if status == RESULT_FAILED and trigger.result.status == RESULT_OK:
                 trigger.result.status = status
+            # in case hysteresis has status 1, but everything is ok, we shloud
```
Comment (benethor) [inline alamo_alerter/alerter.py]: typos in both lines

Comment (h00dy) [inline alamo_alerter/alerter.py]: Fixed.

**Comment by benethor** [comment]
:+1:

**Pull request merged**